src/main/java/compal/commons/Compal.java

```diff
@@ -0,0 +1,157 @@
+package compal.commons;
+
+import compal.model.tasks.Task;
+import compal.storage.StorageManager;
+import compal.ui.UiPart;
+import compal.logic.parser.ParserManager;
+import compal.storage.Storage;
+import compal.model.tasks.TaskList;
+import javafx.scene.paint.Color;
+
+import java.util.ArrayList;
+import java.util.Calendar;
+import java.util.Comparator;
+import java.util.Date;
+
+
+/**
+ * Main class.
+ */
+public class Compal {
+
+    //***Class Properties/Variables***--------------------------------------------------------------------------------->
+    //objects supporting COMPal.Compal
+    public UiPart ui;
+    public Storage storage;
+    public TaskList tasklist;
+    public ParserManager parser;
+    //----------------------->
+
+    //***CONSTRUCTORS***------------------------------------------------------------------------------------------------
+    //------------------------------------------------------------------------------------------------------------------
+    //----------------------------------------------------------------------------------------------------------------->
```

Comment:
Avoid doing this. Try to strictly follow the coding standards for commenting.
https://oss-generic.github.io/process/codingStandards/CodingStandard-Java.html#comments